poppunk/2.6.0/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
What do you think about adding `set -euxo pipefail` somewhere near the top of this script? It makes the output more verbose, but it helps show exactly what is happening in the tests & makes them more robust if there were to be a failure

Reply:
Done! And good to know, as I am not familiar with automated testing.